lib/xml/parser.js

```diff
@@ -234,8 +242,8 @@ function parseEvents(xml) {
         });
 
         formatted.sort((a, b) => {
-            const aStart = new ICAL.Time().fromJSDate(a.data.start);
-            const bStart = new ICAL.Time().fromJSDate(b.data.start);
+            const aStart = new ICAL.Time().fromJSDate(new Date(a.data.start));
+            const bStart = new ICAL.Time().fromJSDate(new Date(b.data.start));
```

Comment:
Why do we have to convert these to `ICAL.Time`? Can't we just compare the JS dates?